lib/api/src/sys/tunables.rs

```diff
@@ -206,6 +206,11 @@ mod tests {
         ) -> Result<VMTable, String> {
             VMTable::from_definition(ty, style, vm_definition_location)
         }
+
+        // Will use a very small stack size of 16kb, not the 1Mb default
+        fn vmconfig(&self) -> &wasmer_vm::VMConfig {
```

Comment:
Can we also export `VMConfig` in `api/src/lib` in the `vm` mod? (that way we can do the following:

```suggestion
        fn vmconfig(&self) -> &crate::vm::VMConfig {
```